melatonin/implementations/vImage.h

```diff
@@ -41,7 +41,10 @@ namespace melatonin::blur
         vImage_Buffer src = { copyData.getLinePointer (0), h, w, (size_t) data.lineStride };
 
         vImage_Buffer dst = { data.getLinePointer (0), h, w, (size_t) data.lineStride };
+
+        JUCE_BEGIN_IGNORE_WARNINGS_GCC_LIKE ("-Wunguarded-availability-new")
```

Comment:
This relates to https://github.com/sudara/melatonin_blur/issues/13#issuecomment-1869021745 — curious why they are fired as warnings at all, I'm assuming it's because the macOS target (like CMAKE_OSX_DEPLOYMENT_TARGET) is lower than these functions and the compiler just has no way to know that we've guarded them?